src/main/java/org/jenkinsci/plugins/badge/BadgeActionFactory.java

```diff
@@ -16,22 +16,42 @@
 @Extension
 public class BadgeActionFactory extends TransientActionFactory<Job> {
 
+    /**
+     *
+     */
     private final ImageResolver iconResolver;
 
+    /**
+     *
+     * @throws IOException 
+     */
     public BadgeActionFactory() throws IOException {
         iconResolver = new ImageResolver();
     }
 
+    /**
+     *
+     * @return
+     */
     @Override
     public Class<Job> type() {
         return Job.class;
     }
 
+    /**
+     *
+     * @param target
+     * @return
+     */
     @Override
     public Collection<? extends Action> createFor(Job target) {
         return Collections.singleton(new BadgeAction(this,target));
     }
-    
+    /**
```

Comment:
[Checkstyle] WARNING: First sentence of Javadoc is incomplete (period is missing) or not present.